src/telemetry.ts

```diff
@@ -1,151 +1,180 @@
-import * as stackTrace from 'stack-trace';
-import TelemetryReporter from 'vscode-extension-telemetry/lib/telemetryReporter';
-import { Constants, Telemetry } from './types/types';
+import TelemetryReporter from "@vscode/extension-telemetry";
+import * as stackTrace from "stack-trace";
+import { Constants, Telemetry } from "./types/types";
 
-const AppinsightsKey = 'AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217';
+const AppinsightsKey = "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217";
```

Comment:
double quotes - did you use a different formatter for this project?

Reply:
I don't know if it has one. This is just the default from VS code.